src/settings/LanguageLocalization/LanguageLocalization.test.js

```diff
@@ -144,7 +166,24 @@ describe('LanguageLocalization', () => {
         value: userSettings,
       });
 
-      expect(mockSetLocale).toHaveBeenCalledWith('en-GB-u-nu-arab');
+      expect(mockSetLocale).toHaveBeenCalledWith('en-GB-u-nu-latn');
```

Comment:
The numbering system shouldn't be changed when the locale is selected. It should be `en-GB-u-nu-arab`.